.github/workflows/osrm-backend.yml

```diff
@@ -74,7 +74,8 @@ jobs:
             CUCUMBER_TIMEOUT: 20000
             CXXCOMPILER: g++-9
             ENABLE_SANITIZER: ON
-            TARGET_ARCH: x86_64-asan
+            TARGET_ARCH: x86_64-asan-ubsan
+            OSRM_CONNECTION_RETRIES: 30
```

Comment:
Some NodeJs tests became flaky after configuring this sanitizer, because as I understand OSRM startup time has increased and 10 retries we are using by default was not enough, so had to add this option and set it to 30.